### PR TITLE
Document default values for query variables #294

### PR DESCRIPTION
**Pull request by SheW0lf**
@stubailo I added a paragraph to the documentation for default values that can be used in query variables. Please have a look and let me know if this is what you had in mind. Thanks a bunch!

### CODE REVIEW

File: site/learn/Learn-Queries.md [hunk 1]
```diff
@@ -165,6 +165,24 @@ Variable definitions can be optional or required. In the case above, since there
...
+query HeroNameAndFriends($episode: Episode = "JEDI") {
+  hero(episode: $episode) {
+    name
+    friends {
+      name
+    }
+  }
+}
+
+```
```
Comment (wincent) [inline site/learn/Learn-Queries.md]: nit: blank line here

Comment (SheW0lf) [inline site/learn/Learn-Queries.md]: Ok I'll fix it :] Does the rest of it look ok? I just want to make sure I'm not missing anything.

Comment (wincent) [inline site/learn/Learn-Queries.md]: Looks pretty good to me. The next sentence after this one is a little confusing though.

**Comment by stubailo** [comment]
Thanks a lot!

**Comment by stubailo** [comment]
Congrats on your first contribution!

**Pull request merged**